src/OpenLoco/src/Windows/CompanyWindow.cpp

```diff
@@ -993,7 +1006,7 @@ namespace OpenLoco::Ui::Windows::CompanyWindow
         {
             World::TileManager::mapInvalidateSelectionRect();
             Input::resetMapSelectionFlag(Input::MapSelectionFlags::enable);
-            auto placementArgs = getHeadquarterPlacementArgsFromCursor(x, y);
+            auto placementArgs = getHeadquarterPlacementArgsFromCursor(x, y, true);
```

Comment:
I'd prefer we didn't pass a bool argument. Just force set the build immediate on the returned arguments here.